social-team.md

```diff
@@ -40,6 +40,7 @@ The basic expectations are that **all** Social Team members will have both a wil
 The Social Team will be responsible for and maintain the following:
 
 - [nodejs/tweet](https://github.com/nodejs/tweet): a repo leveraging [Twitter Together](https://github.com/gr2m/twitter-together) that enables project members to queue up tweets to [@nodejs](https://twitter.com/nodejs) from within GitHub.
+- [nodejs/social-team](https://github.com/nodejs/social-team): a repo for general discussion, logging, and broad requests made of the Social Team.
```

Comment:
Optional nit: Move this above `tweet` so they are in alphabetical order. Doesn't matter much for a list of two items but it's good to have consistency about that sort of thing anyway.

Reply:
Done